### PR TITLE
WiP: Refactor the admin container to be immutable

### PR DESCRIPTION
**Pull request by zioproto**
From Kubernetes docs:
https://kubernetes.io/docs/concepts/containers/

Containers are intended to be stateless and immutable: you should not change the code of a container that is already running. If you have a containerized application and want to make changes, the correct process is to build a new image that includes the change, then recreate the container to start from the updated image.

### CODE REVIEW

File: admin/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 
+#ENV NODE_ENV=production
+#RUN npm ci --only=production ; npm run build # Review package.json for promotion of devDependencies
+RUN npm install
 # Expose port 3000 and build + start application
 EXPOSE 3000
-CMD ["/bin/bash", "-c", "npm install; npm run dev"]
+# Run npm as our non-root user
+
+#CMD ["/bin/bash", "-c", "npm run start"] # Review package.json for promotion of devDependencies
```
Comment (zioproto) [inline admin/Dockerfile:19]: @antonioru I tried to build a version of the `Dockerfile` for production to run `npm run start` but I was not able to. Do you have a working example ?

Comment (zioproto) [reply admin/Dockerfile:19]: I am fine keeping the container running in development mode, but at least let's run an immutable development container :)

I am packaging the cat for Kubernetes, but if the code is mounted at runtime I cant make it work.

please ignore the commented notes about production settings, how do you feel with the rest of the changes ? :) 

Thanks

Comment (antonioru) [reply admin/Dockerfile:19]: Lookin proper sick fam, mad respect ☺️
Appreciate your contribution

**Comment by pieroit** [comment]
Too soon for this, closing the PR.
Thanks

**Pull request closed without merging**